packages/app/obojobo-document-json-parser/src/actionButtonNodeParser/actionButtonNodeParser.js

```diff
@@ -0,0 +1,72 @@
+const textGroupParser = require('../textGroupParser')
+const xmlEncode = require('../xmlEncode')
+
+const actionButtonNodeParser = node => {
+    const id = node.id ? ` id="${node.id}"` : ''
+
+    let attrs = ''
+    for (const attr in node.content) {
+        if ([attr] == "triggers" || [attr] == "textGroup" || [attr] == "actions") continue;
```

Comment:
in many of these parsers, the code `[attr] == "triggers"` should be simplified to `attr == "triggers"

Reply:
Also, I'd like to use a blacklist for the things that must be removed:

```
   const blackListedAttributes = ['triggers', 'scoreActions', 'rubric']

    let attrs = ''
    for (const attr in node.content) {
        if(blackListedAttributes.includes(attr)) continue;
        attrs += ` ${[attr]}="${xmlEncode(node.content[attr])}"`
    }
```

But that leads to the idea of extracting this into a reusable util common to all the parsers so that we would end up with something like `const attrs = processAttrs(node.content, blacklist)`